examples/cryptocurrency/Cargo.toml

```diff
@@ -18,12 +18,18 @@ circle-ci = { repository = "exonum/exonum" }
 
 [dependencies]
 exonum = { version = "0.9.0", path = "../../exonum" }
+exonum_derive = { version = "0.9.0", path = "../../exonum_derive" }
 failure = "0.1.2"
 serde = "1.0.0"
 serde_derive = "1.0.0"
 serde_json = "1.0.0"
+protobuf = "=2.2.0"
```

Comment:
Is strict version number necessary?

Reply:
I suppose not

Reply:
So I think we should remove strict versions if it's possible

Reply:
I guess it's actual for crates with version less then 1.0

Reply:
Ok, I'll remove it in the batch, not in this PR.